src/Layout.ts

```diff
@@ -2276,7 +2150,7 @@ export class BitField {
 
   /** Store a value into the corresponding subsequence of the containing
    * bit field. */
-  decode(b?: Uint8Array, offset?: number): any {
+  decode(b?: Uint8Array, offset?: number): unknown {
```

Comment:
Using `unknown` is debatable -- `BitField` returns a number, but `Boolean` returns a boolean. `any` would work but is loose